.github/workflows/pr-comment.yml

```
@@ -0,0 +1,62 @@
+            let body = `Download the artifacts for this pull request:\n`;
+            for (const art of artifacts) {
+              body += `\n* [${art.name}.zip](https://nightly.link/${owner}/${repo}/actions/artifacts/${art.id}.zip)`;
+            }
+
+            core.info("Review thread message body:", body);
+
+            for (const pr of pull_requests) {
+              body += `\n [Espressif-IDE Documentation Preview](https://preview-docs.espressif.com/projects/espressif-ide/en/${pr.number}-merge/index.html)`;
+              await upsertComment(owner, repo, pr.number,
+                "nightly-link", body);
+            }
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)